### PR TITLE
Add warning enum to CloudPCOnPremisesConnectionStatus

### PR DESCRIPTION
**Pull request by RuiHou105**

### CODE REVIEW

File: api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md [hunk 1]
```diff
@@ -47,7 +47,9 @@ The result of a cloud PC on-premises connection health check.
...
 |adJoinCheckAccessDenied|The active domain join check failed because access is denied when non-administrator users who have been delegated control try to join computer objects to a domain controller. Please assign the correct permission to the customer to join computer object to the domain. Permissions needed: Create computer objects, Delete computer objects.|
 |adJoinCheckUnknownError|The active domain join check failed due to an unknown error. Please contact customer support.|
-|endpointConnectivityCheckUrlNotWhitelisted|The endpoint connectivity check failed because the URLs are not on the allowlist in the network firewall settings. Please add the URLs to the allowlist for the network firewall settings. See [required URL list](/azure/virtual-desktop/safe-url-list) for URL information.|
+|endpointConnectivityCheckCloudPcUrlNotAllowListed|The endpoint connectivity check failed because the CPC provision script storage URL isn’t on the allow list in the network firewall settings. Please add URLs to whitelisted network firewall settings. The URLs can be found in additional information.|
```
Comment (Lauragra) [inline api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md:50]: Please don't use the term "whitelisted"; instead, rephrase i.e. "Add URLs to the list of allowed network firewall settings."

**Review by AshleyYangSZ — APPROVED** [review]

**Comment by RuiHou105** [comment]
1. Related api.md: https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/8271

2. Related CSDL file (schema-Prod-beta.csdl): https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/4415017

**Review by cohanley — APPROVED** [review]

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 2]
```diff
@@ -36,6 +36,56 @@
         },
         {
             "ChangeList": [
+                {
+                    "Id": "e80f229d-0926-443c-be63-67dcbf00e7d0",
+                    "ApiChange": "Member",
+                    "ChangedApiName": "cloudPcOnPremisesConnectionStatus",
```
Comment (isvargasmsft) [inline changelog/Microsoft.CloudManagedDesktop.json:42]: ChangedApiName would be "warning", in this case.

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 3]
```diff
@@ -36,6 +36,56 @@
...
+                    "ChangeType": "Addition",
+                    "Description": "Added the `warning` member to the **cloudPcOnPremisesConnectionStatus** enumeration.",
+                    "Target": "cloudPcOnPremisesConnectionStatus"
+                },
+                {
+                    "Id": "e80f229d-0926-443c-be63-67dcbf00e7d0",
+                    "ApiChange": "Member",
+                    "ChangedApiName": "cloudPcOnPremisesConnectionHealthCheckErrorType",
```
Comment (isvargasmsft) [inline changelog/Microsoft.CloudManagedDesktop.json:50]: ChangedApiName: "endpointConnectivityCheckUrlNotWhitelisted"

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 4]
```diff
@@ -36,6 +36,56 @@
...
+                    "ChangeType": "Deletion",
+                    "Description": "Removed the `endpointConnectivityCheckUrlNotWhitelisted` member from the **cloudPcOnPremisesConnectionHealthCheckErrorType** enumeration.",
+                    "Target": "cloudPcOnPremisesConnectionHealthCheckErrorType"
+                },
+                {
+                    "Id": "e80f229d-0926-443c-be63-67dcbf00e7d0",
+                    "ApiChange": "Member",
+                    "ChangedApiName": "cloudPcOnPremisesConnectionHealthCheckErrorType",
```
Comment (isvargasmsft) [inline changelog/Microsoft.CloudManagedDesktop.json:58]: Similar observassion. Please note, the ChangedAPIName parameter represents the actual element being added, deleted or changed. In this case: endpointConnectivityCheckCloudPcUrlNotAllowListed.

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 5]
```diff
@@ -36,6 +36,56 @@
...
+                    "ChangeType": "Addition",
+                    "Description": "Added the `endpointConnectivityCheckCloudPcUrlNotAllowListed` member to the **cloudPcOnPremisesConnectionHealthCheckErrorType** enumeration.",
+                    "Target": "cloudPcOnPremisesConnectionHealthCheckErrorType"
+                },
+                {
+                    "Id": "e80f229d-0926-443c-be63-67dcbf00e7d0",
+                    "ApiChange": "Member",
+                    "ChangedApiName": "cloudPcOnPremisesConnectionHealthCheckErrorType",
```
Comment (isvargasmsft) [inline changelog/Microsoft.CloudManagedDesktop.json:66]: ChangedApiName: endpointConnectivityCheckWVDUrlNotAllowListed

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 6]
```diff
@@ -36,6 +36,56 @@
...
+                    "ChangeType": "Addition",
+                    "Description": "Added the `endpointConnectivityCheckWVDUrlNotAllowListed` member to the **cloudPcOnPremisesConnectionHealthCheckErrorType** enumeration.",
+                    "Target": "cloudPcOnPremisesConnectionHealthCheckErrorType"
+                },
+                {
+                    "Id": "e80f229d-0926-443c-be63-67dcbf00e7d0",
+                    "ApiChange": "Member",
+                    "ChangedApiName": "cloudPcOnPremisesConnectionHealthCheckErrorType",
```
Comment (isvargasmsft) [inline changelog/Microsoft.CloudManagedDesktop.json:74]: ChangedApiName: endpointConnectivityCheckIntuneUrlNotAllowListed

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
@RuiHou105 thank you very much for your work in this PR. Very nice work. 

Please fix the ChangedApiName parameter in your changelog entries. Please note, this parameter represents the actual element being added, changed or deleted. For details: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog

File: changelog/Microsoft.CloudManagedDesktop.json (hunk 4 above)
Comment (RuiHou105) [reply changelog/Microsoft.CloudManagedDesktop.json]: Thank you for your clear clarifying. I've updated these ChangedApiName. Please have a look. Thanks.

**Review by isvargasmsft — APPROVED** [review]
Good for me!

Thanks!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**